docset/winserver2022-ps/appx/Get-AppSharedPackageContainer.md

```diff
@@ -0,0 +1,101 @@
+﻿---
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-Help.xml
+Module Name: appx
+online version: https://docs.microsoft.com/powershell/module/appx/dismount-appxvolume?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+---
+
+# Get-AppSharedPackageContainer
+
+## SYNOPSIS
+Gets information about the shared package container.
+
+## SYNTAX
+
+```
+Get-AppSharedPackageContainer [[-Name] <String>] [[-Id] <String>] [-AllUsers] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AppSharedPackageContainer cmdlet shows information about any shared package container.
+In particular, it will show what packages are inside the shared package container.
```

Comment:
```suggestion
The cmdlet shows information about any shared package container.
In particular, it will show what packages are inside the shared package container.
```